src/hotspot/share/utilities/devirtualizer.inline.hpp

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Since these new files are mainly copied code, would you mind retaining the copyright dates. Right now this look like newly created code.

Reply:
I updated the copyright year to 2018-2022, since the Devirtualizer class was first added in 2018 in [JDK-8204540](https://bugs.openjdk.org/browse/JDK-8204540)